### PR TITLE
Updated PR template

### PR DESCRIPTION
**Pull request by tondef001**
### Summary

Our PR template is out of date, yo

### Checklist

NO CHECKS FOR YOU

### How to Review

1. Look at it

### Links

none

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -7,10 +7,9 @@ linked together.
 
 ### Checklist
 
-- [ ] Did you run `npm test`?
+- [ ] Have you added unit and functional tests as appropriate?
```
Comment (thofmann) [inline .github/PULL_REQUEST_TEMPLATE.md]: How can we add functional tests when we don't have the functional tests set up yet?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -7,10 +7,9 @@ linked together.
...
 - [ ] Did you update/add documentation for new methods or changed functionality?
-
-Finally, is it ready to be reviewed and merged: yes :white_check_mark: / no :red_circle:
+- [ ] Have you merged the target branch down into this one?
```
Comment (thofmann) [inline .github/PULL_REQUEST_TEMPLATE.md]: There's already a message on PRs that haven't done this. Still probably good to have the checkbox.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -23,6 +22,5 @@ Finally, is it ready to be reviewed and merged: yes :white_check_mark: / no :red
 
 ### Links
 
-Put links to Jira tickets, and anything else, here.
-
 - [TFF-29](https://msoese.atlassian.net/browse/TFF-29)
+- [Sister PR on Consus-Client](https://github.com/TheFourFifths/consus-client/pull/33)
```
Comment (thofmann) [inline .github/PULL_REQUEST_TEMPLATE.md]: We should try to avoid sister PRs as much as possible.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (tondef001) [inline .github/PULL_REQUEST_TEMPLATE.md]: "as appropriate".  THE POINT is to remind people to write all necessary tests.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (tondef001) [inline .github/PULL_REQUEST_TEMPLATE.md]: 👍

**Pull request merged**